src/ansys/fluent/core/services/field_data.py

```diff
@@ -1522,7 +1523,12 @@ def get_mesh(self, zone: str | int) -> Mesh:
         nodes_request = FieldDataProtoModule.GetSolverMeshNodesRequest(
             domain_id=ROOT_DOMAIN_ID, thread_id=zone_info._id
         )
-        nested_nodes = self._service.get_solver_mesh_nodes(nodes_request)
+        # TODO: Add precision query in AppUtilities service
```

Comment:
:+1:

Reply:
@mkundu1 I propose that the `FieldData` service itself should also include a precision() method. This would provide a more complete and self-contained interface for `FieldData`, enabling its reuse in different contexts as well as simplifying client-side logic.